src/Excel/Manager.php

```diff
@@ -113,6 +114,19 @@ public function read(PHPExcel_Worksheet $worksheet, Table $table, array $options
         return $entities;
     }
 
+    private function getCellValue($cell, $column, $options)
```

Comment:
I'm following the CakePHP convention with private/protected methods, so this should be `protected`.